DIFFERENCES.md

```diff
@@ -0,0 +1,82 @@
+In all other cases, ACCP will just return `false` from `Signature.verify()` when given an invalid signature.
+This is different from the default OpenJDK implementation which will also inspect the inner structure of RSA signatures to reject them with a `SignatureException` if they are improperly encoded.
+ACCP follows the guidance provided in [PKCS #1 section 8.2.2](https://tools.ietf.org/html/rfc8017#section-8.2.2) in that it does not parse the inner structure but instead does a binary comparison against the expected value.
+
+For this reason, regardless of whether you use ACCP or not, we recommend the following structure for signature verification:
+```java
+    Signature signatureObject = Signature.getInstance(SIGNATURE_ALGORITHM);
+    signatureObject.initVerify(publicKey);
+    signatureObject.update(messageToVerify);
+    boolean signatureValid = false;
+    try {
+        signatureValid = signatureObject.verify(signature);
+    } catch (final SignatureException ex) {
+        signatureValid = false;
+    }
+```
+
+## Elliptic Curve KeyPairGeneration by curve size
+Neither the JCE nor the default OpenJDK provider for Elliptic Curve Cryptography (SunEC) specify behavior for when `KeyPairGenerator.initialize(int keysize)` is called with an arbitrary value.
+Behavior is only fully specified for the values of 192, 224, 256, 384, and 521.
+This means that applications cannot depend on receiving a specific curve for any other value and may encounter compatibility issues should SunEC ever change its behavior or the application changes to a different JCE provider.
+ACCP removes this unspecified behavior by rejecting use of this method for all values not on the above list (for any ACCP version after 1.5.0) with an `InvalidParameterException`.
+
+For this reason, regardless of whether you use ACCP or not, we recommend that you only use the [KeyPairGenerator.initialize(AlgorithmParameterSpec params)](https://docs.oracle.com/javase/8/docs/api/java/security/KeyPairGenerator.html#initialize-java.security.spec.AlgorithmParameterSpec-) method with an [ECGenParameterSpec](https://docs.oracle.com/javase/8/docs/api/java/security/spec/ECGenParameterSpec.html) to generate EC keys.
+This construction is safe for all known JCE providers and is expected to remain safe even should providers change behavior in other ways.
+
+For more information, please see the [changelog](./CHANGELOG.md) notes for version 1.5.0.
+
+## Cipher.getOutputSize() for AES-GCM
+ACCP may overestimate the amount of space needed when encrypted with `AES/GCM/NoPadding`.
+While this is compliant with the JCE (which [permits overestimation](https://docs.oracle.com/javase/8/docs/api/javax/crypto/Cipher.html#getOutputSize-int-)) it has caused confusion for some developers.
+We are tracking this as [issue #135](https://github.com/corretto/amazon-corretto-crypto-provider/issues/135) and will improve this behavior.
```

Comment:
I wouldn't promise anything. (Anton differs.)

Reply:
That issue is already closed and the awaiting the next release. So, the change has been made and is on the tip of `develop`. It just isn't in the released version yet. I'm comfortable in this case.